scripts/docker-compose.yml

```diff
@@ -0,0 +1,109 @@
+name: ecopal
+
+services:
+  # Cassandra Database container to store the embeddings
+  cassandra2:
+    image: cassandra:latest
+    hostname: cassandra2
+    networks:
+      - backend
+    ports:
+      - "9042:9042"
+    volumes:
+      - cassandra2_data:/var/lib/cassandra/data
+
+  # Postgress Database container used for langflow
+  postgres:
+    image: postgres
+    environment:
+      POSTGRES_USER: langflow
+      POSTGRES_PASSWORD: langflow
+      POSTGRES_DB: langflow
+    ports:
+      - "5432:5432"
+    volumes:
+      - langflow_postgres:/var/lib/postgresql/data
+    networks:
+      - backend
+
+  # Langflow container
+  langflow:
+    image: langflowai/langflow:latest
+    ports:
+      - "7860:7860"
+    depends_on:
+      - postgres
+    environment:
+      - LANGFLOW_DATABASE_URL=postgresql://langflow:langflow@postgres:5432/langflow
+      # This variable defines where the logs, file storage, monitor data and secret keys are stored.
+      - LANGFLOW_CONFIG_DIR=/app/config
+      - LANGFLOW_DO_NOT_TRACK=true
+    volumes:
+      - langflow_data:/app/langflow
+      - langflow_config:/app/config
+      - rag_data:/app/ragdata
```

Comment:
Thanks Karthi but, not needed

Reply:
Are we sure, don't we even need the config folder? Are the saved projects stored in database? if so, we may not need the config folder.

Reply:
Removed langflow_data:/app/langflow and rag_data:/app/ragdata